pkg/server/server.go

```diff
@@ -268,10 +272,7 @@ func NewServer(logger log.Logger, c *Config, handler http.Handler) (*Server, err
 		TLSConfig:         tlsConfig,
 		ReadHeaderTimeout: 30 * time.Second,
 		ReadTimeout:       30 * time.Second,
-		// use flags on standard logger to align with base logger and get consistent parsed fields form adapter:
-		// use shortfile flag to get proper 'caller' field (avoid being wrongly parsed/extracted from message)
-		// and no datetime related flag to keep 'ts' field from base logger (with controlled format)
-		ErrorLog: stdlog.New(log.NewStdlibAdapter(logger), "", stdlog.Lshortfile),
+		ErrorLog:          slog.NewLogLogger(logger.Handler(), slog.LevelError),
```

Comment:
Since we are using slog, the library provides a function that convert a `slog.Logger` to `log.Logger`, including the attributes that we defined.

An example can be found here ->>> [slog2log](https://go.dev/play/p/zA2a8RdNakT)

Reply:
Cool, I didn't know that :)